dbt/include/athena/macros/utils/ddl_data_type.sql

```diff
@@ -0,0 +1,25 @@
+{# Athena has different types between DML and DDL #}
+{# ref: https://docs.aws.amazon.com/athena/latest/ug/data-types.html #}
+{% macro ddl_data_type(col_type) -%}
+    -- transform varchar
+  {% set re = modules.re %}
+  {% set ns = namespace(ddl_type=col_type) %}
+  {% set matching_strings = re.findall('(varchar\(\d+\)|character varying\(\d+\))', ns.ddl_type) %}
```

Comment:
Why did you remove the re.sub implementation ? 
Furthermore, it seems that this implementation is not covering an issue we had about varchar without length.

This should work.
```
{% set data_type = re.sub('(?:varchar|character varying)(?:\(\d+\))?', 'string',  col_type) %}
```

Reply:
```python
>>> x = 'map(map(character varying(22), varchar), map(character varying, varchar(244)))'
>>> re.sub('(?:varchar|character varying)(?:\(\d+\))?', 'string',  x)
'map(map(string, string), map(string, string))'
```

Reply:
Thank you, I forget use updated version of iceberg_helper :bow:
fixed [fe25801](https://github.com/dbt-athena/dbt-athena/pull/30/commits/fe25801da1018af106b5acb747f5825eda72ae84)